### PR TITLE
Cypress tests for header menu page

### PR DESCRIPTION
**Pull request by avneet-raj**

### CODE REVIEW

File: tests/cypress/integration/newfold-labs/wp-module-onboarding/Design/header-menu.cy.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    it('Check if Navigation Next button is visible and go one step next', () => {
+        cy.get('.navigation-buttons_next')
+            .should('be.visible')
+            .click();
+        cy.wait(1000);
+        cy.url().should('not.contain', '/wp-setup/step/design/header-menu');
+    });
+
+    after(() => {
+        cy.clearCustomerData();
```
Comment (officiallygod) [inline tests/cypress/integration/newfold-labs/wp-module-onboarding/Design/header-menu.cy.js:69]: Please check if `cy.setCustomerData();` is even being set on line 6

File: tests/cypress/integration/newfold-labs/wp-module-onboarding/Design/header-menu.cy.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+describe('Header menu Page', function () {
+
+    before(() => {
+        cy.exec('npx wp-env run cli wp theme activate yith-wonder');
+        cy.exec('npx wp-env run cli wp option update mm_brand BlueHost');
```
Comment (officiallygod) [inline tests/cypress/integration/newfold-labs/wp-module-onboarding/Design/header-menu.cy.js:5]: We can go ahead without mm_brand too as that is gonna reduce time in running this Cypress Test.

Comment (avneet-raj) [reply tests/cypress/integration/newfold-labs/wp-module-onboarding/Design/header-menu.cy.js:5]: sure

File: tests/cypress/integration/newfold-labs/wp-module-onboarding/Design/header-menu.cy.js (hunk 1 above)
Comment (avneet-raj) [reply tests/cypress/integration/newfold-labs/wp-module-onboarding/Design/header-menu.cy.js:68]: Done

**Comment by officiallygod** [comment]
Merged!

**Comment by officiallygod** [comment]
https://github.com/officiallygod/bluehost-wordpress-plugin/pull/17

**Comment by officiallygod** [comment]
PR: https://github.com/officiallygod/bluehost-wordpress-plugin/pull/17

**Pull request closed without merging**